CONTRIBUTING.md

```diff
@@ -18,8 +18,8 @@ when submitting issues:
 3. Update to the latest releases of Ember.js, jQuery, Ember Data if possible
 4. Include as much information as possible - this includes full stack traces
   etc.
-5. Set up a demo that demonstrates the issue on [JSFiddle](http://jsfiddle.net)
-  or [JSBin](http://jsfiddle.net). If your project is open source, provide a
+5. Set up a demo that demonstrates the issue on [JSFiddle](https://jsfiddle.net/)
+  or [JSBin](https://jsbin.com/). If your project is open source, provide a
```

Comment:
Not sure whether it's actually possible to use ESA on https://ember-twiddle.com but if so that would be an even better alternative of course.